### ISSUE TITLE
Please add a time picker

### ISSUE THREAD

**Issue by probar** [Pokechu22/Reddit-search-tool#1]
In addition to the date picker, a time picker would be really useful.

**Comment by Pokechu22** [comment]
I agree, I just A) haven't found a good javascript time picker, and B) don't quite know how to handle time zones.

**Comment by probar** [comment]
In that case i think it would be better to split the time entry into 2 options, according to use cases:
1. People want submissions between certain dates - offer a time picker like now. 
2. People want submissions in the last X hours/minutes - offer the appropriate thing. Heck even just a text selection box of how many hours (0-23) would be great. 

I think that covers 95% of the use cases and should work well, do you agree ?
